Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -241,30 +234,26 @@ public void onResult(@Nullable List<PaymentMethodNonce> vaultedPaymentMethods, @
         }
     }
 
-    private void onDismissVaultManager(DropInEvent event) {
-        showSelectPaymentMethodFragment();
+    private void onAddCardSubmit(DropInEvent event) {
+        String cardNumber = event.getString(DropInEventProperty.CARD_NUMBER);
+        showCardDetailsFragment(cardNumber);
     }
 
-    private void showSelectPaymentMethodFragment() {
+    private void showSelectPaymentMethodParentFragment() {
         FragmentManager fragmentManager = getSupportFragmentManager();
-        Fragment fragment = fragmentManager.findFragmentByTag("SELECT_PAYMENT_METHOD");
+        Fragment fragment = fragmentManager.findFragmentByTag("PARENT_FRAGMENT");
```

Comment:
Let's extract these tags into static strings

Reply:
Done! Went ahead and made the fragment transaction code more dry as well.